chapter-09/recipe-06/fortran-example/account/implementation/fortran_implementation.f90

```diff
@@ -1,91 +1,91 @@
 module account_implementation
```

Comment:
Out of curiosity: did you use [fprettify](https://github.com/pseewald/fprettify) or did you re-indent by hand?

Reply:
Using some vim trickery so it took 10 seconds. Did not use a tool.